### PR TITLE
Dis 2.39

### PR DESCRIPTION
**Pull request by spetebroodt**

### CODE REVIEW

File: develop/TOOLS/DIS/XML_editor.md [hunk 1]
```diff
@@ -127,10 +127,12 @@ This editor allows you to design the parameter tables of a protocol using simple
 
 ### Publish
 
-If you click *Publish*, the protocol XML file or the Automation script XML file you are currently editing will be published to the DMA that was set as default DMA in the *DMA* tab of the *DIS Settings* dialog box.
+If you click *Publish*, the protocol or the Automation script you are currently editing will be published to the DMA that was set as default DMA in the *DMA* tab of the *DIS Settings* dialog box.
 
 To publish a protocol or an Automation script to another, non-default DMA, click the drop-down button at the right of the *Publish* button, and click the DMA to which you want the file to be published.
 
+When you publish a protocol or an Automation script, DIS will create either a *.dmprotocol* package (in case of a protocol) or a *.dmapp* package (in case of an Automation script) in the background, install that package on the DataMiner Agent, and then automatically remove it again. This way, there is no need to manually copy all required DLL files on the DataMiner Agent in question.
```
Comment (SimonV-Skyline) [inline develop/TOOLS/DIS/XML_editor.md:134]: At the end of this section, I would add an example of what we are talking about with "DLL files".
Same as done on the "develop/TOOLS/DIS/Developing/Developing_Automation_scripts_as_Visual_Studio_solutions.md" chapter ->
(e.g. DLL files of NuGet packages that are used in the protocol or automation script)

File: release-notes/DIS/DIS_2.39.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+### IDE
+
+#### Enhanced NuGet support [ID_33544] [ID_34113] [ID_34115] [ID_34117]
+
+Up to now, in QAction and EXE projects, DIS would add references to the DataMiner DLL files of the locally installed DataMiner Agent or, if no DataMiner Agent was installed on the local machine, to copies of those files shipped with the DIS installation package. Instead, DIS will now add a reference to a DataMiner DevPack, i.e. a NuGet package that contains the core DataMiner DLL files of a specific DataMiner version. As a separate DataMiner DevPack is available for every released DataMiner version, adapting a solution to a specific DataMiner version is now merely a question of making sure the solution links to the DevPack of that version.
+
+The DataMiner DevPacks can be found on the [official NuGet store](https://www.nuget.org/packages?q=skylinecommunications).
+
+When you open a protocol solution or an Automation script solution in which the QAction and EXE projects still contain references to separate DLL files instead of a DataMiner DevPack, a banner will appear, giving you the opportunity to replace those references by references to the required DevPacks.
```
Comment (SimonV-Skyline) [inline release-notes/DIS/DIS_2.39.md:17]: Maybe add DataMiner here to be clear on what DLLs we are talking about.
Cause it's still allowed to have references to 3rd party DLLs for example. It's only for DataMiner core DLLs that it should now always be done via devpacks.
 "still contain references to separate **DataMiner** DLL Files"

**Review by SimonV-Skyline — COMMENTED** [review]
See my 2 small comments.
Other than that, all good to go to me.

File: develop/TOOLS/DIS/XML_editor.md (hunk 1 above)
Comment (spetebroodt) [reply develop/TOOLS/DIS/XML_editor.md]: Thanks. Example added.

File: release-notes/DIS/DIS_2.39.md (hunk 2 above)
Comment (spetebroodt) [reply release-notes/DIS/DIS_2.39.md]: Thanks. Added.

**Review by SimonV-Skyline — APPROVED** [review]

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**